Formula/watchman.rb

```diff
@@ -27,9 +26,27 @@ class Watchman < Formula
   depends_on "pcre"
   depends_on "python@3.9"
 
+  on_linux do
+    depends_on "gcc"
+  end
+
+  fails_with gcc: "5"
+
+  resource "googletest" do
+    url "https://github.com/google/googletest/archive/release-1.10.0.tar.gz"
+    sha256 "9dc9157a9a1551ec7a7e43daea9a694a0bb5fb8bec81235d8a1e6ef64c716dcb"
+  end
+
   def install
+    resource("googletest").stage do
+      cmake_args = std_cmake_args.reject { |s| s["CMAKE_INSTALL_PREFIX"] }
+      system "cmake", ".", *cmake_args, "-DCMAKE_INSTALL_PREFIX=#{buildpath}/googletest"
+      system "make", "install"
+    end
+
     system "cmake", "-S", ".", "-B", "build",
                     "-DBUILD_SHARED_LIBS=ON",
+                    "-DCMAKE_PREFIX_PATH=#{buildpath}/googletest",
```

Comment:
Let's try `ENV.append_path "CMAKE_PREFIX_PATH"`; I don't think we want to override the env variable set by superenv.

Reply:
I'll try updating after Linux CI results are available to see if any changes are needed there.

Reply:
Actually, I don't think it overrides the value.

For example, in https://cmake.org/cmake/help/latest/command/find_package.html, this should behave similar to `ENV.prepend_path` since it is step 2 **(Search paths specified in cmake-specific cache variables)** while Homebrew superenv uses step 3 **(Search paths specified in cmake-specific environment variables).**

Reply:
For more specific option, I think setting 
```
ENV["GTest_DIR"] = ENV["GMock_DIR"] = buildpath/"googletest"
```
worked for me.

Giving this a try on CI instead of general search path.